extension/src/context.ts

```diff
@@ -0,0 +1,27 @@
+import * as vscode from "vscode";
+import * as nls from "vscode-nls";
+import * as Resources from "./resources";
+import { Disposable } from 'vscode-languageclient';
+import { DocumentManager } from './documents';
+
+
+export class ContextManager{
+	private _ctx: vscode.ExtensionContext;
+	private _docManager: DocumentManager;
+	
+	localize = nls.config({ messageFormat: nls.MessageFormat.file })();	
+	
+	get Context(): vscode.ExtensionContext { return this._ctx; }
```

Comment:
Okay,
Should we expand the DevREADME.md to include a Style Guide section so that is documented?

Reply:
This is the best suggestions!
And I also notice that you use underscore prefix for member class vars. I think we need an suggested coding style guide in the DevREADME.md.

Reply:
I do tend to use the underscore versions for private variables. Definitively something I am used to doing from C# and to be honest makes it pretty clear its for internal use only. I would be slightly concerned about using something like "workspace" without the underscore because I think that may be confusing.

Regarding style guides, we should really start from something that already exists and manipulate it from there. Below are 2 wildly different examples, which do you think would be a better starting point for us?

Minimalist TypeScript Style Guide Example (could be injected into the DevReADME.md)
https://github.com/shahtab/jupyterlab/wiki/TypeScript-Style-Guide

In Depth TypeScript Style Guide (to big, should be a different document)
https://github.com/Platypi/style_typescript

Reply:
I like this one https://github.com/shahtab/jupyterlab/wiki/TypeScript-Style-Guide It is so simple, and easy to follow.

If the style guide is too complex itself, developer only follows the remembered rules.